engine/install/ubuntu.md

```diff
@@ -232,6 +232,7 @@ download a new file each time you want to upgrade Docker Engine.
    `hello-world` image:
 
    ```console
+   $ sudo service docker start
```

Comment:
Kept it the same as the `debian` variant, but wondering if we should change this to `sudo systemctl start docker` (as most of our docs assume systemd is used, and `systemctl` is likely the canonical command for that).

Reply:
should be safe to use systemctl, I think. Can't recall receiving any requests to update this, unlike apt/apt-get. When we get around to defining linux/unix/shell conventions, we could consider including a preference to either command there

Reply:
Yup; let's stick to this for now to keep them the same.

I saw some other things we could update (e.g. the examples for versions listed by `apt-cache madison`, and some other bits and bops).